### ISSUE TITLE
prerelease jobs on debian don't work

### ISSUE THREAD

**Issue by tfoote** [ros-infrastructure/ros_buildfarm#311]
From: [This thread](http://discourse.ros.org/t/run-pre-release-jobs-locally-for-arm-architectures/351/2?u=tfoote)

```
+ echo # BEGIN SECTION: Build Dockerfile - generating devel tasks
# BEGIN SECTION: Build Dockerfile - generating devel tasks
+ cd /tmp/prerelease_job/docker_generating_dockers
+ python3 -u /tmp/prerelease_job/ros_buildfarm/scripts/misc/docker_pull_baseimage.py
Get base image name from Dockerfile 'Dockerfile': ubuntu:jessie
Check docker base image for updates: docker pull ubuntu:jessie
Pulling repository docker.io/library/ubuntu
Tag jessie not found in repository docker.io/library/ubuntu
```

The backtrace suggests that devel jobs might not work either.

**Comment by dirk-thomas** [comment]
I was able to successfully run a prerelease job for Debian using the following command:

```generate_prerelease_script.py https://raw.githubusercontent.com/ros-infrastructure/ros_buildfarm_config/production/index.yaml kinetic default debian jessie amd64 genmsg --level 0 --output-dir ./```

I would assume the problem with Debian devel / prerelease jobs has been addressed some time ago (#302). Can you please post an example command which doesn't work?

**Comment by tfoote** [comment]
The last release ([1.1.0](https://github.com/ros-infrastructure/ros_buildfarm/tree/1.1.0)) was before #302 was merged so it probably just needs a new release.
